demo-simple/build.gradle

```diff
@@ -28,11 +28,13 @@ android {
 }
 
 dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])
-    androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation 'com.novoda:notils:2.2.16'
+    implementation 'com.facebook.stetho:stetho:1.5.0'
```

Comment:
The two above are cloned from `library build.gradle` didn't expose with `api` because we probably don't want to expose it to clients of the library. We use these for debugging only.